### PR TITLE
add seeds

### PR DESCRIPTION
**Pull request by MStodolkiewicz**
Added seeds for Movies, Halls, Seances

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,64 @@
...
+h3 = Hall.create(name: 'Hall 3', capacity: 50)
+h4 = Hall.create(name: 'Hall 4', capacity: 50)
+h5 = Hall.create(name: 'Hall 5', capacity: 50)
+h6 = Hall.create(name: 'Hall 6', capacity: 100)
+h7 = Hall.create(name: 'Hall 7', capacity: 100)
+h8 = Hall.create(name: 'Hall 8', capacity: 100)
+h9 = Hall.create(name: 'Hall 9', capacity: 100)
+h10 = Hall.create(name: 'Hall 10', capacity: 200)
+
+m1 = Movie.create(name: 'Harry Potter', description: 'Wizard movie', duration: 154)
```
Comment (h4ppyr0gu3) [inline db/seeds.rb]: Please look into faker gem 
it will save a lot of time instead of writing out this data

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,64 @@
...
+m10 = Movie.create(name: 'Avengers: Infinity War', description: 'Marvel movie', duration: 134)
+
+s1 = Seance.create(movie_id: 1, hall_id: 10, start_time: Time.now + 2.days, price: 21)
+s2 = Seance.create(movie_id: 2, hall_id: 9, start_time: Time.now + 3.days, price: 19)
+s3 = Seance.create(movie_id: 3, hall_id: 8, start_time: Time.now + 4.days, price: 17)
+s4 = Seance.create(movie_id: 4, hall_id: 7, start_time: Time.now + 1.days, price: 20)
+s5 = Seance.create(movie_id: 5, hall_id: 6, start_time: Time.now + 5.days, price: 16)
+s6 = Seance.create(movie_id: 6, hall_id: 5, start_time: Time.now + 3.days, price: 18)
+s7 = Seance.create(movie_id: 7, hall_id: 4, start_time: Time.now + 2.days, price: 22)
+s8 = Seance.create(movie_id: 8, hall_id: 3, start_time: Time.now + 5.days, price: 14)
```
Comment (h4ppyr0gu3) [inline db/seeds.rb]: same as above

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,64 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+h1 = Hall.create(name: 'Hall 1', capacity: 20)
```
Comment (h4ppyr0gu3) [inline db/seeds.rb]: there is no need to assign these to variables if they are  not called

File: db/seeds.rb (hunk 1 above)
Comment (MStodolkiewicz) [reply db/seeds.rb]: Thanks! I installed faker gem and created loops instead of single records :)

File: db/seeds.rb [hunk 4]
```diff
@@ -1,68 +1,27 @@
...
-s33 = Seance.create(movie_id: 1, hall_id: 5, start_time: Time.now + 9.days + 20.minutes, price: 10)
-s34 = Seance.create(movie_id: 1, hall_id: 2, start_time: Time.now + 3.days + 20.hours, price: 10)
-s35 = Seance.create(movie_id: 1, hall_id: 4, start_time: Time.now + 5.days + 10.hours, price: 10)
-s36 = Seance.create(movie_id: 1, hall_id: 9, start_time: Time.now + 5.days + 1.hours, price: 10)
-s37 = Seance.create(movie_id: 1, hall_id: 10, start_time: Time.now + 5.days + 5.hours, price: 10)
-s38 = Seance.create(movie_id: 1, hall_id: 8, start_time: Time.now + 5.days + 7.hours, price: 10)
-s39 = Seance.create(movie_id: 1, hall_id: 7, start_time: Time.now + 5.days + 19.hours, price: 10)
+500.times do
+    Seance.create(
+        movie_id: rand(10) + 1,
```
Comment (baniak66) [inline db/seeds.rb]: I think `Movie.all.sample.id` would be more elegant 😉 Same with the `hall_id`.

File: db/seeds.rb [hunk 5]
```diff
@@ -1,68 +1,27 @@
...
+        hall_id: rand(10) + 1,
+        start_time: rand(14).days.from_now + rand(24).hours + rand(60).minutes,
+        price: rand(10) + 15
```
Comment (baniak66) [inline db/seeds.rb:25]: Why `+ 15`?

File: db/seeds.rb (hunk 4 above)
Comment (MStodolkiewicz) [reply db/seeds.rb]: Yup, looks better, thanks :).

File: db/seeds.rb (hunk 5 above)
Comment (MStodolkiewicz) [reply db/seeds.rb:25]: It's not really necessary, just for the price to look about right.

**Pull request merged**